test/e2e/framework/deployment/deployment.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2017 The Kubernetes Authors.
+Copyright 2019 The Kubernetes Authors.
```

Comment:
Don't update copy right date if you're moving a file

Reply:
Will keep in mind. thank you!